src/VssDatabase.cpp

```diff
@@ -203,6 +202,22 @@ jsoncons::json tryParse(jsoncons::json val) {
         throw outOfBoundException(msg.str());
       }
     } else if (value_type == "boolean") {
+      string v=val.as<string>();
+      boost::algorithm::erase_all(v, "\"");
+
+      if ( v == "true") {
```

Comment:
would we like to also support "True", "TRUE"?

Reply:
I checked and according to https://www.w3schools.com/js/js_json_datatypes.asp   only "true" and "false" in JSON, so I elft it at that. (and made the error message explicitly mentioning that)